discord/guild.py

```diff
@@ -895,6 +896,65 @@ def get_member(self, user_id: int, /) -> Member | None:
         """
         return self._members.get(user_id)
 
+    _FETCHABLE = TypeVar(
```

Comment:
I think using the `bound` parameter will fix the issue mentioned in your comment, @TheGiga.

Reply:
Sadly, it doesn't fix the issue i mentioned, but it made code more compact